src/main/resources/synthea.properties

```diff
@@ -23,7 +23,8 @@ exporter.hospital.fhir_dstu2.export = false
 exporter.practitioner.fhir.export = true
 exporter.practitioner.fhir_stu3.export = false
 exporter.practitioner.fhir_dstu2.export = false
-exporter.csv.export = false
+exporter.practitioner.preserve_uuid = true
+exporter.csv.export = true
```

Comment:
Please do not change the default setting of `exporter.csv.export`. It should be `false` by default.

Reply:
fixed